src/Orc.Automation/Win32/WindowsMessages.cs

```diff
@@ -0,0 +1,225 @@
+﻿namespace Orc.Automation.Win32;
+
+public static class WindowsMessages
```

Comment:
Do we really need all these messages? And I think itt should be called "WindowMessages" (singular)